res/css/views/emojipicker/_EmojiPicker.pcss

```
@@ -224,6 +224,9 @@ limitations under the License.
 
 .mx_EmojiPicker_preview_text {
     display: flex;
+    width: 78%;
```

I don't think we should use a hardcoded value like this as if the emoji picker size changes, this won't flow automatically.

The parent of `mx_EmojiPicker_preview_text` is a flex container we can therefore make the container grow to take the space available, and use the overflow property to force the text to wrap

```suggestion
    flex: 1;
    overflow: hidden;
```